spec/services/laa_reference_updater_spec.rb

```diff
@@ -6,7 +6,10 @@
   let(:contract) { NewLaaReferenceContract.new.call({ maat_reference: maat_reference, defendant_id: defendant_id }) }
   let(:mock_record_laa_reference_service) { double Api::RecordLaaReference }
```

Comment:
perhaps this can be an [`instance_double`](https://relishapp.com/rspec/rspec-mocks/v/3-2/docs/verifying-doubles/using-an-instance-double)?
```suggestion
  let(:mock_record_laa_reference_service) { instance_double Api::RecordLaaReference }
```

Reply:
Fixed!